bootstrap_cfn/fab_tasks.py

```diff
@@ -496,6 +495,30 @@ def get_zone_id():
                 "for zone name '%s'...", zone_id, zone_name)
     return zone_id
 
+@task
+def get_hosted_name():
+    cfn_config = get_config()
+
+    r53_conn = get_connection(R53)
+    import pdb;
+    for elb in cfn_config.data['elb']:
```

Comment:
you may want to check if cfn_config.data has KeyError when 'elb' is not in yaml by adding try and except, or cfn_config.data.get('elb') returns none if the key doesn't exist.